editor/src/clj/util/debug_util.clj

```diff
@@ -58,6 +63,17 @@
               (DurationFormatUtils/formatDuration ms "m:ss")
               (DurationFormatUtils/formatDuration ms "H:mm:ss"))))))))
 
+(defn bytes->string
+  "Converts a byte size into a human-readable size string."
+  ^String [^long bytes]
+  (let [double-bytes (double bytes)]
+    (condp > double-bytes
+      bytes-per-kilobyte (str bytes " B")
+      bytes-per-megabyte (decimal-string (/ double-bytes bytes-per-kilobyte) "KB")
```

Comment:
It is very good that you used "KB" instead of "kB" to avoid confusion. Because [people use](https://en.wikipedia.org/wiki/Byte#Multiple-byte_units) e.g. "MB" to mean both `1000*1000 bytes` and `1024*1024 bytes` depending on the context. Which is completely normal and not confusing at all, because there is "KB" vs "kB" to differentiate different things...